src/one_dragon/utils/app_utils.py

```
@@ -12,25 +11,47 @@ def start_one_dragon(restart: bool) -> None:
     :param restart: 是否重启
     :return: 是否成功
     """
-    launcher_path = os.path.join(os_utils.get_work_dir(), 'OneDragon-Launcher.exe')
+    if getattr(sys, 'frozen', False):
+        launcher_path = Path(sys.executable)
+    else:
+        launcher_path = Path(os_utils.get_work_dir()) / 'OneDragon-Launcher.exe'
     subprocess.Popen(f'cmd /c "start "" "{launcher_path}""', shell=True)
```

Ruff (0.15.2)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)